File: 240206/CHJmission01.c
#include <stdio.h>
#include <stdlib.h>
#include <wiringPi.h>
#include <unistd.h>
#include <time.h>
#define LED_RED 7
#define LED_GREEN 21
#define LED_BLUE 22
int main(void){
        int i;
        if(wiringPiSetup () == -1)
                return 1;
        pinMode(LED_RED,OUTPUT);
        pinMode(LED_GREEN,OUTPUT);
        pinMode(LED_BLUE,OUTPUT);
        digitalWrite(LED_RED,0);
        digitalWrite(LED_GREEN,0);
        digitalWrite(LED_BLUE,0);
        printf("3 Color LED Control Start !! \n");
         for(i=0;i<10;i++){
                printf("LED On !! \n");
                digitalWrite(LED_RED,1);
                digitalWrite(LED_GREEN,1);
                digitalWrite(LED_BLUE,1);	  
                usleep(500000);
                printf("LED Off !! \n");
                digitalWrite(LED_RED,0);
                digitalWrite(LED_GREEN,0);
                digitalWrite(LED_BLUE,0);
                usleep(500000);
        }
        return 0;
}

File: 240206/CHJmission01.py
import RPi.GPIO as GPIO
import time

LED_RED = 4  # GPIO 핀 번호
LED_GREEN = 5  # GPIO 핀 번호
LED_BLUE = 6  # GPIO 핀 번호

def main():
    GPIO.setmode(GPIO.BCM)
    GPIO.setup(LED_RED, GPIO.OUT)
    GPIO.setup(LED_GREEN, GPIO.OUT)
    GPIO.setup(LED_BLUE, GPIO.OUT)

    GPIO.output(LED_RED, GPIO.LOW)
    GPIO.output(LED_GREEN, GPIO.LOW)
    GPIO.output(LED_BLUE, GPIO.LOW)

    print("3 Color LED Control Start !!")

    for i in range(10):
        print("LED On !!")
        GPIO.output(LED_RED, GPIO.HIGH)
        GPIO.output(LED_GREEN, GPIO.HIGH)
        GPIO.output(LED_BLUE, GPIO.HIGH)
        time.sleep(0.5)

        print("LED Off !!")
        GPIO.output(LED_RED, GPIO.LOW)
        GPIO.output(LED_GREEN, GPIO.LOW)
        GPIO.output(LED_BLUE, GPIO.LOW)
        time.sleep(0.5)

    GPIO.cleanup()

if __name__ == "__main__":
    main()
